tasks/main.yml

```diff
@@ -34,9 +40,9 @@
         name: "{{ item }}"
         source: pull
       loop:
-        - cisagov/guacscanner:1.1.6
-        - guacamole/guacd
-        - guacamole/guacamole
+        - cisagov/guacscanner:1.1.11
+        - guacamole/guacd:1.4.0
+        - guacamole/guacamole:1.4.0
         # The version of the JDBC PostgreSQL driver included in the
```

Comment:
Not related to this PR, but since it caught my eye, did you happen to check if the new Guacamole image contains an updated JDBC PostgreSQL driver?

https://github.com/cisagov/ansible-role-guacamole/blob/d6b9328139413cd5f109034320b96a0655445cdf/tasks/main.yml#L46-L50

Reply:
I did mean to check on that but forgot, so thanks for the reminder.  It looks like [version 1.4.0 of Guacamole is still using an ancient JDBC PostgreSQL driver](https://github.com/apache/guacamole-client/blob/1.4.0/guacamole-docker/bin/build-guacamole.sh#L111), but [the driver is updated to the most recent in the `master` branch](https://github.com/apache/guacamole-client/blob/master/guacamole-docker/bin/build-guacamole.sh#L111).

I have created cisagov/guacamole-composition#37 to track this work.  See also commit fa228ca.

Reply:
Bummer- https://github.com/apache/guacamole-client/pull/655 with this change was merged into `master` on Dec. 4, 2021, but it wasn't included in [1.4.0](https://github.com/apache/guacamole-client/releases/tag/1.4.0) which was tagged on Jan. 2, 2022.